File: pyround1/1470.重新排列数组.py
#
# @lc app=leetcode.cn id=1470 lang=python3
#
# [1470] 重新排列数组
#

# @lc code=start
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        rs = []
        for i in range(n):
            rs.append(nums[i])
            rs.append(nums[i+n])
        return rs
# @lc code=end

File: 1470.重新排列数组.cpp
/*
 * @lc app=leetcode.cn id=1470 lang=cpp
 *
 * [1470] 重新排列数组
 */

// @lc code=start
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> res;
        int start = 0;
        while(start < n){
            res.push_back(nums[start]);
            res.push_back(nums[start+n]);
            start++;
        }
        return res;
    }
};
// @lc code=end

